src/Compilers/CSharp/Test/Semantic/Semantics/MethodBodyModelTests.cs

```diff
@@ -170,11 +170,11 @@ public void OvldRslnWithExplicitIfaceImpl()
         {
             var text = @"
 using System;
-interface i1
+interface @i1
```

Comment:
Why this is required? The character `1` isn't in the `[a-z]` range.

Reply:
Good catch. Those indeed can be left unescaped.

Reply:
Note that the `#Resolved` and similar edits are inserted into our comments by an internal tool. If the goal is to match the conventions of that tool, then:
- we commonly use the tag `#Closed` to indicate that the original author of the comment has confirmed the issue to be addressed
- the tag `#Resolved` indicates the PR author has made changes to address someone's comment, but the original author of the comment must confirm that the issue was addressed before changing the state to `#Closed`.